txros/src/txros/serviceclient.py

```diff
@@ -8,62 +8,66 @@
 
 
 class ServiceError(Exception):
+
     def __init__(self, message):
         self._message = message
+
     def __str__(self):
         return 'ServiceError(%r)' % (self._message,)
     __repr__ = __str__
 
+
 class ServiceClient(object):
+
     def __init__(self, node_handle, name, service_type):
         self._node_handle = node_handle
         self._name = self._node_handle.resolve_name(name)
         self._type = service_type
-    
+
     @util.cancellableInlineCallbacks
     def __call__(self, req):
         serviceUrl = yield self._node_handle._master_proxy.lookupService(self._name)
-        
+
         protocol, rest = serviceUrl.split('://', 1)
         host, port_str = rest.rsplit(':', 1)
         port = int(port_str)
-        
+
         assert protocol == 'rosrpc'
-        
-        conn = yield endpoints.TCP4ClientEndpoint(reactor, host, port).connect(util.AutoServerFactory(lambda addr: tcpros.Protocol()))
+
+        conn = yield endpoints.TCP4ClientEndpoint(reactor, host, port).connect(
+            util.AutoServerFactory(lambda addr: tcpros.Protocol()))
         try:
             conn.sendString(tcpros.serialize_dict(dict(
                 callerid=self._node_handle._name,
                 service=self._name,
                 md5sum=self._type._md5sum,
                 type=self._type._type,
             )))
-            
-            header = tcpros.deserialize_dict((yield conn.receiveString()))
-            # XXX do something with header
-            
+
+            tcpros.deserialize_dict((yield conn.receiveString()))
```

Comment:
I removed the assignment of `header` here because it was not used. I figured `conn.receiveString()` was still necessary, though.